src/main/resources/jupyter/init-actions.sh

```diff
@@ -153,11 +156,15 @@ if [[ "${ROLE}" == 'Master' ]]; then
     gsutil cp ${JUPYTER_DOCKER_COMPOSE} /etc
     gsutil cp ${RSTUDIO_DOCKER_COMPOSE} /etc
     gsutil cp ${PROXY_DOCKER_COMPOSE} /etc
+    gsutil cp ${WELDER_DOCKER_COMPOSE} /etc
 
     # Needed because docker-compose can't handle symlinks
     touch /hadoop_gcs_connector_metadata_cache
     touch auth_openidc.conf
 
+    # do not enable welder yet.  Remove flag when welder is complete.
+    WELDER_ENABLED=false
```

Comment:
DO NOT enable Welder's management of the Jupyter /work dir yet.  Merging this should not affect Jupyter functionality.

Reply:
If you want to enable it, ssh to the master node and:
```
sudo -s
gcloud docker --authorize-only
export WELDER_DOCKER_IMAGE=us.gcr.io/broad-dsp-gcr-public/welder-server:5c3d6bb
docker-compose -f /etc/welder-docker-compose.yaml up -d
```

Reply:
do you also need to `export WELDER_ENABLED=true`?

Reply:
no. it is only used in this script.

Reply:
ah!